docs/imb.md

```diff
@@ -10,6 +10,4 @@ Before running the pipeline, you will need to load Nextflow using the environmen
 module load nextflow
 ```
 
-:::note
-You will need an account to use the IMB HPC cluster in order to run the pipeline. If in doubt contact IT.
-:::
+> You will need an account to use the IMB HPC cluster in order to run the pipeline. If in doubt contact IT.
```

Comment:
```suggestion
:::note
You will need an account to use the IMB HPC cluster in order to run the pipeline. If in doubt contact IT.
:::
```

Wait this is a rendering bug on the website. I will fix this, please leave it in 😬

Reply:
Alright, thanks @mashehu!

Whilst you're at it, could you maybe take a look at the [parsing](https://github.com/nf-core/website/blob/ac12d95396c25ac2c74f0490debe9336687c2871/src/pages/configs/%5Bconfigslug%5D.astro#L116) of the `config_profile_contact` param?

Currently the website sometime links to unrelated github profiles if `config_profile_contact` contains an email address instead of a github handle. Examples [here](https://nf-co.re/configs/cambridge) and [here](https://nf-co.re/configs/imb). Probably requires some regex that distinguishes email addresses from gh handles.